### DISCUSSION TITLE
Iphones unable to choose column headers

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#279] [Q&A]
Currently, to change column headers in EME, we need to right click or long press to bring up the column header selection pane. This works fine for desktops and Android phones. However, for Iphones, my EME users are facing a problem whereby they are unable to choose the column headers because for some reason, long-pressing the column headers does not bring up the column header selection pane.

Any solution for this issue?

**Comment by liedekef** [comment]
The admin interface is not really meant for smartphones, but I'm guessing on iphones you need to use another browser for it to work ... I should switch to something else maybe, but changing jtable to another lib would take a lot of time.

**Reply by refugeindhamma** [reply]
They tried Safari and Chrome and both can't work. Will changing to using Wordpress "Screen Option" also take a lot of time?

**Reply by liedekef** [reply]
Wordpress screen options are *very* limited and take a long time to implement. I did that in the past, but the simplistic way they implement dynamic tables is just not cutting it ...

**Reply by refugeindhamma** [reply]
Ok. Hope you can implement a solution soon. I realized that the column header setting does not transfer to other devices or even a different cookie session even if logged into the same account. My EME users are using their mobile phones to scan the QR codes for attendance and they need to be able to see the attendance count in order to send emails to absentees from the "Approved Bookings" page.

Alternatively, is there a way to permanently change the column headers such that all EME users can see the change?
